Composer/packages/client/src/utils/hooks.ts

```diff
@@ -50,10 +50,17 @@ export const useLinks = () => {
 };
 
 export const useRouterCache = (to: string) => {
+  const projectId = useRecoilValue(currentProjectIdState);
```

Comment:
![image](https://user-images.githubusercontent.com/39758135/98217356-5dd0ab80-1f85-11eb-8754-23f00dfa8f80.png)
currentProjectId looks like the active bot id